src/main/java/at/aau/se2/handler/game/subhandlers/RequestUsernamesForSwitchHandler.java

```diff
@@ -0,0 +1,52 @@
+package at.aau.se2.handler.game.subhandlers;
+
+import at.aau.se2.handler.game.GameHandler;
+import at.aau.se2.utils.Lobby;
+import at.aau.se2.utils.Player;
+import at.aau.se2.utils.UtilityMethods;
+import com.fasterxml.jackson.databind.JsonNode;
+import org.springframework.web.socket.TextMessage;
+import org.springframework.web.socket.WebSocketSession;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+
+public class RequestUsernamesForSwitchHandler extends RequestUsernamesHandler{
+
+    ArrayList<String> usernames = new ArrayList<String>();
+    @Override
+    public void handleMessage(WebSocketSession session, JsonNode msg, Lobby lobby) {
+        try{
+            List<Player> players = GameHandler.getPlayersofGame();
```

Comment:
@TimonWoelbitsch 
ACHTUNG: Hier passiert eine Referenzübergabe => Username würde aus der Main Player Liste gelöscht werden
```suggestion
            List<Player> players = List.copyOf(GameHandler.getPlayersofGame());
```